docs/releases/1.11.txt

```diff
@@ -205,7 +205,9 @@ Database backend API
 Miscellaneous
 -------------
 
-* ...
+* The :meth:`~django.utils.feedgenerator.SyndicationFeed.latest_post_date` now
```

Comment:
I'd write this as 
```:meth:`SyndicationFeed.latest_post_date <django.utils.feedgenerator.SyndicationFeed.latest_post_date>```
otherwise it just renders as "latest_post_date" which isn't obvious as to what it refers to.